src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -227,6 +233,15 @@ public static String fixAuthorFirstNameFirstCommas(String authors, boolean abbr,
         return AuthorList.parse(authors).getAsFirstLastNames(abbr, oxfordComma);
     }
 
+    /**
+     * This is a convenience method for getAuthorsFirstFirstLatexFree()
+     *
+     * @see AuthorList#getAsFirstLastNames
+     */
+    public static String fixAuthorFirstNameFirstCommasLatexFree(String authors, boolean abbr, boolean oxfordComma) {
```

Comment:
I'm not a big fan of the `fix...` methods (I know you didn't introduced them) and would propose to inline all of them. With a bit of refactoring and extracting `AuthorList.parse(authors)` this shouldn't introduce that much more code elsewhere.